contribs/gmf/src/directives/print.js

```diff
@@ -136,7 +139,22 @@ gmf.module.directive('gmfPrint', gmf.printDirective);
  */
 gmf.PrintController = function($scope, $timeout, $q, $injector, gettextCatalog,
     ngeoLayerHelper, ngeoFeatureOverlayMgr,  ngeoPrintUtils, ngeoCreatePrint,
-    gmfPrintUrl, gmfAuthentication) {
+    gmfPrintUrl, gmfAuthentication, ngeoQueryResult, ngeoFeatureHelper,
+    $filter) {
+
+
+  /**
+   * @type {angular.$filter}
+   * @private
+   */
+  this.$filter_ = $filter;
```

Comment:
```
/**
 * @type {function(string): string}
 * @private
 */
this.translate_ = $filter('translate');
```

And later use `this.translate_(s)` instead of `this.$filter_('translate')(s);`